### DISCUSSION TITLE
Using this LeoCAD for Erector sets or better yet GRIDBEAM

### DISCUSSION THREAD

**Discussion by mdklegin** [leozide/leocad#892] [Ideas]
So were do one start to import building blocks into LeoCAD? I found this really interesting building system for real world things like vehicles, houses and furniture using perforated metal or wood conduit and would love to be able to use LeoCAD as it dont require to beefy of a machine to design models etc.  here is some sauce on gridbeam. https://youtu.be/PIMESt9iLYg?feature=shared

**Comment by leozide** [comment]
You have to create your own library and point LeoCAD to it, check out ldraw.org for the file format.

**Comment by MrEdgeNL** [comment]
Hi mdklegin, 
I'm also creating [custom parts](https://retrobuildingtoys.nl/leocad/) for LeoCAD.
Also created some [documentation](https://drive.google.com/file/d/1plB4Dx51sGTxYOGS2yjA4ScP405dIldP/view?usp=drive_link), maybe usefull for you.

Gridbeam looks a lot like [Matador](https://www.matador.at/Products/Explorer-5/Explorer-E717::184.html).
I might add Matador to LeoCAD someday too (low priority project)...

Good luck!
